src/fetching/RelayConnectionFetcher.js

```diff
@@ -72,22 +90,24 @@ export default class RelayConnection<T, Data> implements Fetcher<Data[]> {
 		};
 
 		while (retries < this.maxRetries) {
-			const res = await fetch(url, options);
-			const data = await res.json();
-
-			if (data.errors && Array.isArray(data.errors)) {
+			try {
+				const res = await fetch(url, options);
+				const { status } = res;
+				if (retryCodes.includes(status)) {
+					retries = retries + 1;
+					// eslint-disable-next-line no-console
+					console.log(`RelayConnectionFetcher: retry after server response code ${status}`);
+				} else if (exitCodes.includes(status)) {
+					throw new Error(`server response error ${status}`);
+				} else {
+					const data = await res.json();
+					return getConnection(data);
```

Comment:
here you also need to somehow return the errors if you want to use them in `transformResult`

Reply:
the graphQL errors are already inside of `data` (`data.errors`).
So you meant http errors?

Reply:
No, i mean that it would make sense to return sth. like
```javascript
{...getConnection(data), errors: data.errors}
```

Otherwise, all clients need to remember to do that in their `getConnection` implementations